R/run.R

```diff
@@ -870,22 +870,6 @@ create_child_runs <- function(parent_run,
   parent_run$create_children(count, tag_key, tag_values)
 }
 
-#' Submit an experiment and return the active child run
-#'
-#' @description
-#' Submit an experiment and return the active child run.
-#' @param parent_run The parent `Run` object.
-#' @param config The `RunConfig` object
-#' @param tags Tags to be added to the submitted run, e.g., {"tag": "value"}.
-#' @return A `Run` object.
-#' @export
-#' @md
-submit_child_run <- function(parent_run,
```

Comment:
why remove the function? let's try to add support for child runs in estimators in Python SDK.

Reply:
okay, sounds good. In the meantime though, should we un-export it so users don't try using the estimator runconfig (like in the failed test) and get errors?

Reply:
Adding @mx-iao

Reply:
yeah, for now let's unexport the method